ethtool/src/header.rs

```diff
@@ -0,0 +1,126 @@
+// Copyright 2021 Red Hat, Inc.
```

Comment:
Hello @cathay4t, would it be possible to not add these copyright comments? I'm ignorant of copyright law so I feel a bit uncomfortable such headers.

Reply:
Let me remove those lines from source code even that does not changed the fact that Red Hat is owning these code implicitly.
BTW, these copyright line has been exists in kernel and many other projects for years.

Reply:
Removed.

Reply:
@cathay4t  You might miss `ethtool/src/header.rs`.
The copyright lines still exist in that file.

Reply:
Oops. Thanks!